pkg/kubelet/cm/helpers.go

```diff
@@ -65,6 +65,7 @@ func getLibcontainerCgroupManager(cgroupConfig *CgroupConfig) (*cgroupfs.Manager
 	// Extract the cgroup resource parameters
 	resourceConfig := cgroupConfig.ResourceParameters
 	resources := &libcontainerconfigs.Resources{}
+	resources.AllowAllDevices = true
```

Comment:
Actually I don't think we need this line. This line got in by mistake when I was testing the code.
I will remove this line